TaskService/packages.config

```diff
@@ -1,32 +1,32 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr" version="3.4.1.9004" targetFramework="net45" />
-  <package id="bootstrap" version="3.0.0" targetFramework="net45" />
-  <package id="jQuery" version="1.10.2" targetFramework="net45" />
-  <package id="Microsoft.AspNet.Mvc" version="5.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.Razor" version="3.2.0" targetFramework="net45" />
+  <package id="Antlr" version="3.5.0.2" targetFramework="net451" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net451" />
   <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi" version="5.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.HelpPage" version="5.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebPages" version="3.2.0" targetFramework="net45" />
-  <package id="Microsoft.IdentityModel.Logging" version="5.2.1" targetFramework="net451" />
-  <package id="Microsoft.IdentityModel.Protocols" version="5.2.1" targetFramework="net451" />
-  <package id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="5.2.1" targetFramework="net451" />
-  <package id="Microsoft.IdentityModel.Tokens" version="5.2.1" targetFramework="net451" />
-  <package id="Microsoft.Owin" version="4.0.0" targetFramework="net451" />
-  <package id="Microsoft.Owin.Host.SystemWeb" version="4.0.0" targetFramework="net451" />
-  <package id="Microsoft.Owin.Security" version="4.0.0" targetFramework="net451" />
-  <package id="Microsoft.Owin.Security.ActiveDirectory" version="4.0.0" targetFramework="net451" />
-  <package id="Microsoft.Owin.Security.Jwt" version="4.0.0" targetFramework="net451" />
-  <package id="Microsoft.Owin.Security.OAuth" version="4.0.0" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebApi" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebApi.HelpPage" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.7" targetFramework="net451" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net451" />
+  <package id="Microsoft.IdentityModel.JsonWebTokens" version="5.4.0" targetFramework="net451" />
+  <package id="Microsoft.IdentityModel.Logging" version="5.4.0" targetFramework="net451" />
```

Comment:
>< [](start = 2, length = 1)

I do not see the MSAL included in this one. Just making sure its not required. The line , if MSAL was indeed needed by this project would be
  <package id="Microsoft.Identity.Client" version="3.0.8" targetFramework="net461" />

Reply:
The previous version of this sample wasn't using ADAL on this TaskService project. This project basically just add the Todo item in the memory list and list all the Todo's in the list. So no changes needed to be done here.